packages/x-charts/src/models/axis.ts

```diff
@@ -140,6 +140,11 @@ export interface ChartsYAxisProps extends ChartsAxisProps {
 }
 
 export interface ChartsXAxisProps extends ChartsAxisProps {
+  /**
+   * The minimum gap in pixels between two tick labels.
+   * @default 8
```

Comment:
Probably good to mention that if two tick labels overlap, one of them will be hidden.